src/views/projects/SourceBrowser/FileDiff.svelte

```diff
@@ -118,7 +118,7 @@
   <header class="file-header" on:click={collapse}>
     <div class="actions">
       <p class="txt-regular">{file.path}</p>
-      {#if mode === "created"}
+      {#if mode === "added"}
```

Comment:
wanna change the badge to `added` here as well?

Reply:
(github's not very good at rendering this, check `Badge` on the next line)

Reply:
Here you go: 2f27d2abe8eb8d2f15e5ee53c3ee71faabbd4b8b.